rust/arrow/src/ipc/gen/SparseTensor.rs

```diff
@@ -389,7 +389,7 @@ impl<'a> SparseMatrixIndexCSX<'a> {
     }
     /// indptrBuffer stores the location and size of indptr array that
     /// represents the range of the rows.
-    /// The i-th row spans from indptr[i] to indptr[i+1] in the data.
+    /// The i-th row spans from `indptr[i]` to `indptr[i+1]` in the data.
```

Comment:
I was seeing `cargo doc` warnings about these lines just this morning. 👍  for fixing them